src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -258,6 +261,65 @@ private static boolean isPresent(String availableChecks, String checkName) {
         return availableChecks.matches(linkPattern);
     }
 
+    private static Map<String, String> readSummaries() throws Exception {
```

Comment:
Utility methods should go after the method(s) that use them.

Reply:
done